### DISCUSSION TITLE
Case Card

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#204] [Component suggestions]
<img width="480" alt="visuele weergave van het case card component, inclusief heading, datum en melding dat er 1 taak open staat" src="https://github.com/nl-design-system/backlog/assets/545207/ed94d505-aae3-4e39-9097-75725e8d74c7">

We willen graag meer leren over het **Case Card** component. We zien dit als een component om te linken naar een zaak met daarin wat meta-informatie. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht op Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1704894524613379)

---

## Naam
Case Card

## Doel
Toont meta-informatie over een zaak en biedt de mogelijkheid hier naartoe te navigeren.

## Gerelateerde componenten
[Card as link](https://github.com/orgs/nl-design-system/discussions/305)

## Onderzoek
Zie comments.

**Comment by jiromaykin** [comment]
Elke Case Card geef taan welke status bereikt is: per status kan een kleur worden geconfigureerd 
(uit de set van 4: info, succes, warning, error).

In OIP (Open Inwoner platform) gebruiken we Cards niet alleen voor Cases maar ook voor andere data:
Structuren:
https://github.com/maykinmedia/open-inwoner/tree/develop/src/open_inwoner/components/templates/components/Card
https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/components/templatetags/card_tags.py
SASS:
https://github.com/maykinmedia/open-inwoner/blob/develop/src/open_inwoner/scss/components/Card/Card.scss

<img width="1152" alt="cards-OIP-cases" src="https://github.com/nl-design-system/backlog/assets/118177951/4f94f7c1-2467-4ead-a17f-d9500f24297c">

**Reply by MariekeBrouwer** [reply]
[Als aanvulling: https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=9%3A23578&mode=design&t=KfSQWXbv3TURY1pn-1](https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=1-813&mode=design&t=mhPcu9POaosTJFJT-0) hier het specifieke figma file.

**Reply by UXTeamGroningen** [reply]
Nog een aanvulling, op basis van onderzoek.
Er komen bij ons case cards voor op de Homepage en op de Zaakoverzichtspagina, binnen onze Mijn Omgeving.
Uit onderzoek blijkt dat de gebruikte cards op beide pagina's duidelijk zijn. Op de overzichtspagina is het dan meteen duidelijk wat de status is van de aanvraag. En of er dus een actie nodig is. 

We tonen ook ten allen tijde het zaaknummer. Dit is het nummer wat een inwoner 'moet' hebben op het moment dat hij navraag doet bij de 1e lijn telefonie bijvoorbeeld. Zonder het 'zaaknummer' hierbij te zetten was het niet duidelijk waar het over ging. 

https://gebruikersonderzoeken.nl/docs/onderzoek-bekijken/mijn-zaken/groningen-mijn-omgeving-homepage/

**Comment by jeffreylauwers** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-casecard--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/Card)
- [Figma - Huidig](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/%F0%9F%8C%BB-Den-Haag-Design-System?type=design&node-id=1696-16&mode=design&t=t3NfFuAiLFGYjnBs-4)

---

### Screenshots / Varianten

**Active**
![Scherm­afbeelding 2024-01-22 om 11 35 25](https://github.com/nl-design-system/backlog/assets/545207/1e128371-0b0a-49d1-a577-48d3f0203bb0)

**Inactive**
![Scherm­afbeelding 2024-01-22 om 11 35 19](https://github.com/nl-design-system/backlog/assets/545207/a2647dff-38cb-43da-96e1-88430a46b3d0)


---

### Notities
- Optie voor tonen datum.
- Optie voor tonen Status badge.
- Optie voor inactieve status.

---

### Toelichting Rozerin Ayerdem

De case cards worden op dit moment gebruikt in de [Mijn Omgeving van de gemeente Den Haag](https://www.figma.com/proto/jvKCJhmZGKv0fVJxSbxZ8v/Erfpacht---MijnDenHaag-%2F-Amsterdam?page-id=6727%3A555&type=design&node-id=13378-43077&viewport=-30028%2C7668%2C0.61&t=w1uU0T115UnmiAoJ-1&scaling=min-zoom&starting-point-node-id=13378%3A43077&show-proto-sidebar=1&mode=design).

Hieronder volgt een samenvatting van de bevindingen uit het [onderzoek Generieke Services](https://codefornl.slack.com/files/UQAS9M2F2/F0527KEFKTJ/rapport_generieke_services_23032023_def.pdf):

- De cards worden effectief herkend als mappen.
- Het label ‘1 taak open’ functioneert goed. Een respondent gaf aan: “Dit betekent dat er iets moet gebeuren,” terwijl een ander opmerkte: “Oh ja, ik zie dat er nog een taak openstaat. Nou, dan moet ik daar wat mee.”
- We constateren dat gebruikers behoefte hebben aan meer informatie over wat er precies aan de hand is. Zo hebben we een visuele indicator die aangaf dat er een taak openstond vervangen door het label ‘1 taak open’, wat veel beter begrepen wordt. Het principe ‘Don’t make me think’ sluit goed aan bij onze doelgroep.
- Gebruikers uiten twijfels over de getoonde datum, aangezien niet iedereen weet waar deze voor staat. Dit is iets dat we kunnen verbeteren. Er is nooit specifiek onderzoek gedaan naar de vraag of gebruikers überhaupt de datum willen zien. Hoewel wij, ontwerpers, dit handig vinden, met name met het oog op de toekomstige sorteermogelijkheden, zou het zinvol zijn om dit nader te onderzoeken.

En andere aandachtspunten:

- We zijn aan het kijken hoe we ‘veel’ zaken kunnen presenteren. Tonen we bijvoorbeeld nog steeds de case cards of gaan we over naar een ander soort view? En bij welk aantal is dit?
- Op welke manier tonen we de naam van een zaak? Hoe kan een gebruiker een zaak gemakkelijk herkennen? Als je meerdere zaken hebt met dezelfde titel, dan weet je al gauw niet meer bij welke je moet zijn (denk aan: ‘Bezwaar tegen parkeerbon’ -> welke van de 5 parkeerbonnen die ik heb?). Hoe kunnen we een zaaknaam identiek maken zodat het snel vindbaar is?

**Comment by jeffreylauwers** [comment]
## Amsterdam

Maakt gebruik van Case card van Den Haag voor een prototype rondom Erfpacht.
- [Figma - Prototype](https://www.figma.com/file/jvKCJhmZGKv0fVJxSbxZ8v/Erfpacht---MijnDenHaag-%2F-Amsterdam?type=design&node-id=17659-99257&mode=design&t=decpaPm0pqlD8Q8s-4)

---

### Screenshots / Varianten
<img width="480" alt="ams" src="https://github.com/nl-design-system/backlog/assets/545207/dbbc821b-a64b-41e7-a4cb-1b1230b55e56">


---

### Toelichting Nico Druif

- Deze werd heel goed herkend en aangeklikt tijdens recent gebruikersonderzoek.
- Wel nog een uitdaging: wat is het maximum aantal case cards dat we tonen als er veel zaken lopen? Dan switchen naar een tabelview?

Aansluitend op de vraag over de datum, en de naam van de zaak:
- "Over de betekenis van de datum werden aannames gedaan. Hier moet duidelijk staan dat dit de datum van de eerste aanvraag is, dus startdatum van de zaak. "
- "In deze tabel is het handig als het adres is vermeld, het soort aanvraag en de aanvrager/contactpersoon" -> dat hier ook behoefte is aan de aanvrager heeft te maken met het feit dat er meerdere mensen vanuit de organisatie achter de knoppen kunnen zitten. Dus de ene keer een medewerker van de ene afdeling, de andere keer een andere medewerker. Dus men wil zien wie deze zaak heeft gestart.

![Scherm_afbeelding 2024-01-17 om 10 29 53](https://github.com/nl-design-system/backlog/assets/545207/d05eac46-7370-4686-90e7-d8a59547d359)

**Comment by jeffreylauwers** [comment]
## Groningen

Maakt gebruik van een algemeen card component met specifieke invulling voor een zaak. [Zie comment Jiro en Marieke](https://github.com/orgs/nl-design-system/discussions/204#discussioncomment-8095066)

- [Figma](https://www.figma.com/file/iKGhWhstaLIlFSaND2q7cE/OIP---Designs-(new)?type=design&node-id=1-813&mode=design&t=mhPcu9POaosTJFJT-0)

---

### Screenshots / Varianten

**Default**
<img width="384" alt="Scherm­afbeelding 2024-01-22 om 12 32 28" src="https://github.com/nl-design-system/backlog/assets/545207/79b64e40-77ec-4eb5-9c67-d9585dce8d4e">


**Status: Document toevoegen**
<img width="386" alt="Scherm­afbeelding 2024-01-22 om 12 32 21" src="https://github.com/nl-design-system/backlog/assets/545207/43cfb398-d65d-469f-b96d-3283726b7b68">

**Status: Uitslag bekend**
<img width="382" alt="Scherm­afbeelding 2024-01-22 om 12 32 33" src="https://github.com/nl-design-system/backlog/assets/545207/1ad39096-bf83-461a-8457-246a85e5a682">

---

### Notities
- Optie voor tonen Status badge.
- Label call-to-action wijzigt per status.

---

### Toelichting Jana

**Op de vraag of er uit gebruikersonderzoek was gebeleken of er meer data nodig was. En of er ooit onderzocht hoe gebruikers op een weergave met minder data reageren.**

In eerste instantie hadden we nog meer data getoond (zie oude screenshot van een detail pagina van een zaak).

![image (1)](https://github.com/nl-design-system/backlog/assets/545207/7ec38ea5-916e-43a2-860c-889467e393dd)

Hier kwam na voren dat inwoners niet wisten wat de data betekenden en welke data belangrijkst was. Aansluitend aan Nico hebben we daarom de betekenis van de data aangescherpt en andere worden gebruikt b.v. "Beslissing op zijn laats op" en "aanvraag ingediend op" op i.p.v. eindatum en ontvangstdatum. Inwoners wilden in het onderzoek bovendien graag weten wanneer ze uiteindelijk de uitslag krijgen, dus daarom hebben we dit ook benoemd en in de designs opgenomen. In de Figma files/designs is de bewoording volgens mij nog niet verwerkt.

Nog minder data tonen hebben we niet getest omdat inwoners het prettig vonden om een uitslagdatum te weten en wij ervoor gekozen hebben om de zaken/cards op de overzichtspagina op data te sorteren i.p.v. geen filter/sorteer button te tonen. Hoezo hebben we ervoor gekozen?: Een particulier heeft bij ons volgens data gemiddeld maar 5/6 zaken lopen (parkeerboeten lopen b.v. niet over de gemeente bij ons). Meestal worden alle zaken dus op een pagina getoond en is automatisch sorteren op data (meest recente eerst) onze keuze geweest. De screenshot boven van Jiro is een open/niet afgeronde aanvraag, daar communiceren we twee data en geen zaaknummer omdat inwoners de aanvraag nog moeten indienen. Ook hier moet de bewoording  op de de screenshot nog aangepast worden

De feed forward badge is met inwoners getest. De badge verandert ook afhankelijk van de status van de zaak (b.v. upload documenten, bekijk aanvraag of bekijk beslissing). In het verleden zagen we in onderzoek dat inwoners niet begrepen dat ze door klikken op de aanvraag/card op een detail pagina van de zaak terecht kwamen en daar hun documenten kunnen uploaden. Nu is duidelijk voor hun dat ze zo b.v. documenten kunnen uploaden of de uitslag kunnen bekijken. Niemand liep meer tegen dat probleem aan. De flow is dus zeker voor de inwoner verbetert en ook hier is de key duidelijke informatie geven en benoemen wat ze achter te tegel kunnen zien/doen

**Reply by UXTeamGroningen** [reply]
Onderzoek voor de toelichting van Jana: https://gebruikersonderzoeken.nl/docs/onderzoek-bekijken/mijn-zaken/groningen-mijn-omgeving/

**Comment by alimpens** [comment]
De naam Case Card en het doel ('een component om te linken naar een zaak met daarin wat meta-informatie') voelt voor mij nogal specifiek. Is er een reden om niet voor het meer generieke Card te gaan?

**Reply by jeffreylauwers** [reply]
Hey Aram, klopt dit component is ook specifiek bedoelt in dit geval voor een 'zaak systeem'. We verwachten we dat er ook een meer generiek Card component nodig zal zijn. Hiervoor kan iedereen een Discussion starten.

**Comment by jeffreylauwers** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Case card op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=case+card)
